src/geocoder.js

```diff
@@ -8,20 +8,25 @@ var util = require('./util'),
 // Low-level geocoding interface - wraps specific API calls and their
 // return values.
 module.exports = function(url, options) {
-
+    if (!options) options = {};
     var geocoder = {};
 
     util.strict(url, 'string');
 
     if (url.indexOf('/') === -1) {
-        url = urlhelper('/geocode/' + url + '/{query}.json', options && options.accessToken);
+        if (options.proximity) {
+            url = urlhelper('/geocode/' + url + '/{query}.json?proximity={proximity}', options.accessToken);
+        } else {
+            url = urlhelper('/geocode/' + url + '/{query}.json', options.accessToken);
+        }
     }
 
     geocoder.getURL = function() {
         return url;
     };
 
-    geocoder.queryURL = function(_) {
+    geocoder.queryURL = function(_, args) {
```

Comment:
`queryURL ` hasn't been documented as a public API, but I wouldn't be surprised if it's in use somewhere, and so long as we have to use the single argument format for `query`, let's use it for `queryURL` too. Meaning, keep the `queryURL` signature the same, move the type checking of `_` into `queryURL`, and leave `query` doing a straight pass through -- `geocoder.queryURL(_)`.